src/screens/GroupSetting/hooks/useGroupSetting.js

```diff
@@ -4,23 +4,17 @@ import {launchImageLibrary} from 'react-native-image-picker';
 
 import {Linking} from 'react-native';
 import {useNavigation} from '@react-navigation/core';
-import {useRecoilState, atom} from 'recoil';
+import {useRecoilState} from 'recoil';
 import {Context} from '../../../context';
 import {getChatName} from '../../../utils/string/StringUtils';
 import {requestExternalStoragePermission} from '../../../utils/permission';
 import SignedMessageRepo from '../../../service/repo/signedMessageRepo';
 import ChannelList from '../../../database/schema/ChannelListSchema';
 import useLocalDatabaseHook from '../../../database/hooks/useLocalDatabaseHook';
-import {CHANNEL_GROUP, CHAT_ATOM} from '../../../hooks/core/constant';
+import {CHANNEL_GROUP} from '../../../hooks/core/constant';
 import StringConstant from '../../../utils/string/StringConstant';
-import {uploadFile} from '../../../service/file';
-
-const chatAtom = atom({
-  key: CHAT_ATOM,
-  default: {
-    selectedChannel: null
-  }
-});
+import {chatAtom} from '../../../hooks/core/chat/useChatUtilsHook';
+import useUploadImage from '../../../hooks/useUploadImage';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been modified to include the `useUploadImage` hook and the `chatAtom` from `useChatUtilsHook`. The `uploadFile` function is no longer imported, and the local definition of `chatAtom` has been removed. These changes seem appropriate given the context provided.
```diff
- import {useRecoilState, atom} from 'recoil';
+ import {useRecoilState} from 'recoil';
- import {CHANNEL_GROUP, CHAT_ATOM} from '../../../hooks/core/constant';
+ import {CHANNEL_GROUP} from '../../../hooks/core/constant';
- import {uploadFile} from '../../../service/file';
+ import {chatAtom} from '../../../hooks/core/chat/useChatUtilsHook';
+ import useUploadImage from '../../../hooks/useUploadImage';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->